core/services/commander/main.py

```diff
@@ -87,6 +88,18 @@ async def command_host(response: Response, command: str, i_know_what_i_am_doing:
         return {"message": f"{error}"}
 
 
+@app.post("/set_time", status_code=status.HTTP_200_OK)
+@version(1, 0)
+async def set_time(response: Response, unix_time_seconds: int, i_know_what_i_am_doing: bool = False) -> Any:
+    unix_time_seconds_now = int(time.time())
+    if abs(unix_time_seconds_now - unix_time_seconds) < 30 * 60:
```

Comment:
can you think of any reasons not to have a lower number here? 30 minutes is enough time to turn off the rov and move it somewhere else, which would result in some logs with the wrong time. not a great issue, but slightly inconvenient

Reply:
Any time may be arbitrary, what you suggest ? 5min ?

Reply:
Updated to 5min.